benchmarks/vector_db_compare/vectorlite_bench.py

```
@@ -0,0 +1,363 @@
+    configure_sqlite(db, args.page_size, args.cache_mb)
+    count = db.execute("select count(*) from documents").fetchone()[0]
+    if count > 0:
+        payload = db.execute("select document from documents where rowid = 1").fetchone()[0]
+        probe_document = json.loads(bytes(payload))
+        probe = np.asarray(probe_document["embedding"], dtype="<f4").tobytes()
+        probe_rows = db.execute(
+            "select rowid from vectors where knn_search(embedding, knn_param(?, 1, ?))",
+            [probe, args.ef_search],
+        ).fetchall()
+        if len(probe_rows) != 1:
+            raise RuntimeError(f"vectorlite reopen probe returned {len(probe_rows)} rows, want 1")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)